python/setup.py

```diff
@@ -22,7 +22,11 @@
     license = 'Apache License (2.0)',
     packages = ['pywatchman'],
     ext_modules = [
-        Extension('pywatchman.bser', sources = ['pywatchman/bser.c'])
+        Extension('pywatchman.bser', sources = ['pywatchman/bser.c']
+        #    ,
+        #    extra_compile_args=['-O0', '-g'],
```

Comment:
yes and no... it was handy while debugging.  I'll tidy this up and/or remove it before landing